src/postgres/event-handlers/CommentDeleted.ts

```diff
@@ -1,12 +1,12 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
+import { PostId } from '@subsocial/types/substrate/interfaces/subsocial';
 import { deleteNotificationsAboutComment } from '../delete-activity';
 import { deleteCommentFollower } from '../delete-follower';
 import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
 
 export const onCommentDeleted: EventHandlerFn = async (eventAction: SubstrateEvent) => {
   const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const postId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, postId } = parsePostEvent(eventAction);
```